checker/check-request-property-enum-value-updated.go

```diff
@@ -28,24 +28,38 @@ func RequestPropertyEnumValueRemovedCheck(diffReport *diff.Diff, operationsSourc
 				CheckModifiedPropertiesDiff(
 					mediaTypeDiff.SchemaDiff,
 					func(propertyPath string, propertyName string, propertyDiff *diff.SchemaDiff, parent *diff.SchemaDiff) {
-						enumDiff := mediaTypeDiff.SchemaDiff.EnumDiff
-						if enumDiff == nil || enumDiff.Deleted == nil {
-							return
-						}
-						if propertyDiff.Revision.Value.ReadOnly {
+						enumDiff := propertyDiff.EnumDiff
+						if enumDiff == nil {
 							return
 						}
+
 						for _, enumVal := range enumDiff.Deleted {
+							level := ERR
+							if propertyDiff.Revision.Value.ReadOnly {
+								level = INFO
+							}
 							result = append(result, ApiChange{
 								Id:          "request-property-enum-value-removed",
-								Level:       ERR,
+								Level:       level,
 								Text:        fmt.Sprintf(config.i18n("request-property-enum-value-removed"), enumVal, ColorizedValue(propertyFullName(propertyPath, propertyName))),
```

Comment:
Let's use ColorizedValue(enumVal) instead of enumVal

Reply:
using it here will break output as this message doesn't contain ''. I suggest we  defer all those changes to another text improvement PR and flag the potential risk.

Reply:
ok